tests/functional/test_list.py

```diff
@@ -109,7 +109,7 @@ def test_local_columns_flag(simple_script: PipTestEnvironment) -> None:
     assert "Package" in result.stdout
     assert "Version" in result.stdout
     assert "simple (1.0)" not in result.stdout
-    assert "simple     1.0" in result.stdout, str(result)
+    assert "simple  1.0" in result.stdout, str(result)
```

Comment:
This test changes because `pip list --local` does not show `coverage` and `setuptools` while it did before.
Therefore the column is smaller.

I think this is correct because coverage and setuptools are outside of the virtualenv, according to our implementation of `BaseDistribution.local`.